tests/rustapi_module/setup.py

```diff
@@ -0,0 +1,58 @@
+import sys
+
+from setuptools import setup
+from setuptools.command.test import test as TestCommand
+from setuptools_rust import RustExtension
+
+
+class PyTest(TestCommand):
+    user_options = []
+
+    def run(self):
+        self.run_command("test_rust")
+
+        import subprocess
+        errno = subprocess.call(['pytest', 'tests'])
+        raise SystemExit(errno)
+
+
+def get_py_version_cfgs():
+    # For now each Cfg Py_3_X flag is interpreted as "at least 3.X"
+    version = sys.version_info[0:2]
+
+    if version[0] == 2:
+        return ['--cfg=Py_2']
+
+    py3_min = 5
+    out_cfg = []
+    for minor in range(py3_min, version[1]+1):
+        out_cfg.append('--cfg=Py_3_%d' % minor)
+
+    return out_cfg
+
+setup_requires = ['setuptools-rust>=0.6.1']
```

Comment:
`0.6.1` -> `0.10.2`

Reply:
Theoretically we could also get rid of this entirely, since it's preferable to use `pyproject.toml`, though that would mean that `python setup.py test` probably wouldn't work (not sure if it works now).

Reply:
yep, deleting that line is also fine.